lib/appeals/responses/appeals.rb

```diff
@@ -7,8 +7,16 @@ class Appeals < Common::Base
       attribute :status, Integer
 
       def initialize(body, status)
-        self.body = body
+        self.body = body if json_format_is_valid?(body)
         self.status = status
+      rescue JSON::Schema::ValidationError => error
+        raise Common::Client::Errors::ClientError.new(error.message, 500)
```

Comment:
Could the app be more specific about the error? 500s are reserved for when we break things in this case it's not us it's them 😄 (as long as the schema we check against is correct). This is the best [http code decision tree](https://www.codetinkerer.com/2015/12/04/choosing-an-http-status-code.html) I've found, and as the request is good, and the other server is responding, but it's not returning a valid response so it seems like a `Common::Client::Errors::ParsingError` with a status of `502` would work here.